config.json

```diff
@@ -806,6 +806,14 @@
         "topics": [
           "control_flow_loops"
         ]
+      },
+      {
+        "slug": "pig-latin",
+        "name": "Pig Latin",
+        "uuid": "3bee6827-8766-4d4d-9d0f-3facc3bfd3f4",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

Comment:
I think the difficulty should be higher. A common difficulty is somewhere between 4 and 6 which corresponds to a medium difficulty shown on the website.

Reply:
Should be done. Thanks